docs/GettingStarted/SpeedrunTricks.md

```diff
@@ -23,6 +23,8 @@ By some miracle, some of these warps can lead you to the level's goal ring.  It'
 ## Chaos Control Extension
 When using Chaos Control, it will go for a set amount of time, regardless of how much of you hero meter you have left. This trick is specifically for the case where you have already been using Chaos Control for a bit, but canceled just before running out of the meter, thus being able to reactivate before the meter is completely drained.  If you do so at the right time in the right spot, you can go further into a level than just the single Chaos Control could.
 
+Additionally, having no partners out may increase the range of your chaos control if the framerate is adversely affected by having the partner out in that particular level. This varies per area and per level.
```

Comment:
I'd like to see a specific example for this case.  I was recently doing a bit of testing with CC range for different builds of the game for region / consoles and changing the partner character did not seem to help.  At least not noticeably at the time.  However, there is a noticeable difference in framerate when the partner is hidden, so we should add a section on this page for that effect.

Reply:
Lost Impact comes to mind, as having Maria out / not out changes where your drop with CC

Reply:
Seems I was wrong, the difference here is caused by the spline taken

## Hero
![hero](https://user-images.githubusercontent.com/14857235/161391498-e5a35fd4-0eb2-44de-b560-835d67f5a273.jpg)


## Normal
![normal](https://user-images.githubusercontent.com/14857235/161391493-b1215871-43cc-41a0-b023-e69a90148165.jpg)

Reply:
Removed as not valid